### PR TITLE
Narae

### PR DESCRIPTION
**Pull request by narae970210**
Week 3 tutorial

### CODE REVIEW

File: Week3_Introduction_to_Data.ipynb [hunk 1]
```diff
@@ -0,0 +1,3030 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "JEQ9rcFCZV0l",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Data Manipulation with Pandas\n",
+        "This week, we will cover the basic data manipulation using Pandas.\n",
+        "Pandas is an open source data analysis and manipulation tool, built on top of the Python programming.\n",
```
Comment (kylebegovich) [inline Week3_Introduction_to_Data.ipynb:36]:
```suggestion
        "Pandas is an open-source data analysis and manipulation tool, built on top of the Python programming language.\n",
```

File: Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb [hunk 2]
```diff
@@ -3,77 +3,153 @@
...
-        "colab_type": "text"
+        "colab_type": "text",
+        "id": "gJVCL5-e24GK"
+      },
+      "source": [
+        "## Why, Where and How we use Pandas\n",
+        "\n",
+        "This week, we will cover the basic data manipulation using Pandas.\n",
+        "1. Pandas is an open source data analysis and manipulation tool and it is widely used both in academia and industry.\n",
+        "2. It is built on top of the Python programming. \n",
```
Comment (kylebegovich) [inline Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb:90]:
```suggestion
        "2. It is built on top of the Python programming language. \n",
```

File: Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb [hunk 3]
```diff
@@ -3,77 +3,153 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "t17ksTXD4j_c"
+      },
+      "source": [
+        "## What we will be learning today\n",
+        "\n",
+        "### About the dataset:\n",
+        "To begin with Pandas and dataframe, we will use titanic dataset. It has all of the passengers' information, such as name, age, gender, ticket price, and most importantly whether they survived or not.\n",
```
Comment (kylebegovich) [inline Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb:114]:
```suggestion
        "To begin with Pandas and dataframes, we will use a dataset about the Titanic. It has all of the passengers' information, such as name, age, gender, ticket price, and most importantly whether they survived or not.\n",
```

File: Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb [hunk 4]
```diff
@@ -3,77 +3,153 @@
...
+        "\n",
+        "As each person has its unique PassengerID, each row is a unique entity / passenger.\n",
+        "\n",
+        "\n",
+        "\n",
+        "### Goals:\n",
+        "Using data manipulation and analysis, we will learn how to get a quick overview of the dataset, how to select and filter specific rows / columns. We At the end of the tutorial, we should be able to analyze the survival rates."
```
Comment (kylebegovich) [inline Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb:121]:
```suggestion
        "Using data manipulation and analysis, we will learn how to get a quick overview of the dataset, how to select and filter specific rows / columns. At the end of the tutorial, we should be able to analyze the survival rates."
```

File: Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb [hunk 5]
```diff
@@ -3,77 +3,153 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "DMD_T3YbB9s_"
       },
       "source": [
-        "## Connecting to Your Google Drive\n"
+        "## Connecting to Your Google Drive\n",
+        "\n",
+        "We will start by connecting google drive into google colab"
```
Comment (kylebegovich) [inline Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb:133]: getting an error on this cell, `ModuleNotFoundError: No module named 'google'
`

Comment (narae970210) [reply Week3-Introduction-to-Open-Data-Importing-Data-and-Basic-Data-Wrangling/Week3_Introduction_to_Data.ipynb:133]: This code works if you are using Google cola. We haven't decided on which format we are going to use (Github or Google cola) but I will make changes based on that. Thank you for letting me know!

**Pull request merged**